common/components/WalletDecrypt/components/DeterministicWalletsModal.tsx

```diff
@@ -113,21 +122,20 @@ class DeterministicWalletsModalClass extends React.Component<Props, State> {
         handleClose={onCancel}
       >
         <div className="DWModal">
+          {/* TODO: replace styles for flexbox with flexbox classes in https://github.com/MyEtherWallet/MyEtherWallet/pull/850/files#diff-2150778b9391533fec7b8afd060c7672 */}
           <form className="DWModal-path form-group-sm" onSubmit={this.handleSubmitCustomPath}>
-            <span className="DWModal-path-label">Addresses for</span>
-            <select
-              className="form-control"
+            <span className="DWModal-path-label">Addresses </span>
```

Comment:
Did you have any thoughts from the commentary in #828 on how we could make it more clear what this is? Could be good to get in some usability improvements while we're here, but feel free to punt to a future PR.

Reply:
I think it's probably out of scope for now, considering this PR needs a Part 2 on it's own. It'll be good to circle back to address (😉) them though.